app/Http/Controllers/Api/UsersController.php

```diff
@@ -427,13 +427,10 @@ public function show($id) : JsonResponse | array
      * @param  \Illuminate\Http\Request  $request
      * @param  int  $id
      */
-    public function update(SaveUserRequest $request, $id) : JsonResponse
+    public function update(SaveUserRequest $request, User $user): JsonResponse
     {
         $this->authorize('update', User::class);
```

Comment:
I'm assuming this can go away now, and was there to first check that a user could update users in general before doing the check to find the user? but wanted to double check.